fs-storage/Makefile

```
@@ -0,0 +1,52 @@
+CARGO_CMD = cargo
+JAR_FILE := junit.jar
+URL := https://repo1.maven.org/maven2/org/junit/platform/junit-platform-console-standalone/1.9.3/junit-platform-console-standalone-1.9.3.jar
+
+.PHONY: test-linux
+test-linux: build-rust download-test-lib java-test-linux
+
+.PHONY: test-mac
+test-mac: build-rust download-test-lib java-test-mac
+
+.PHONY: test-windows
+test-windows: build-rust download-test-lib java-test-windows
+
+.PHONY: build-rust
+build-rust:
+	$(CARGO_CMD) build
+
+.PHONY: download-test-lib
+download-test-lib: 
+	cd tests && \
```

```suggestion
	@cd tests && \
```

So it doesn't print the command to the terminal each time